src/sdk/namespace_nb.js

```diff
@@ -312,6 +313,34 @@ class NamespaceNB {
     put_object_retention(params, object_sdk) {
         return object_sdk.rpc_client.object.put_object_retention(params);
     }
+
+    ////////////
+    //  ACLs  //
+    ////////////
+
+    async get_object_acl(params, object_sdk) {
+        await object_sdk.read_object_md({
+            bucket: params.bucket,
+            key: params.key,
+            version_id: params.versionId
+        });
+
+        return {
+            owner: s3_utils.DEFAULT_S3_USER,
+            access_control_list: [{
+                Grantee: { ID: s3_utils.DEFAULT_S3_USER, Type: "CanonicalUser" },
+                Permission: "FULL_CONTROL"
+            }]
+        };
+    }
+
+    async put_object_acl(params, object_sdk) {
+        return await object_sdk.read_object_md({
```

Comment:
We should return just `{version_id}` right? So maybe best not to return the entire object_md if this is not how other namespaces work, and do it
like you did for get_object_acl.

Also can you put the todo comment that was in this code :
```
// TODO S3 ignoring put_object_acl for now
```

Reply:
Gotcha